app/views/includes/compareValueInput.scala.html

```diff
@@ -0,0 +1,41 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(
+    previousLabel: String,
+    newLabel: String,
+    newField: Field,
+    previousFieldName: String,
+    previousFieldValue: String
+)(implicit m: Messages)
+
+@import includes.forms.input
+@import uk.gov.hmrc.play.views.helpers.MoneyPounds
+
+<div class="grid-row">
+    <div class="column-half">
+        <div class="form-label form-label-bold">@previousLabel</div>
+        <input type="hidden" id="oldAmount" name='@previousFieldName' value='@previousFieldValue' />
+        <p>£@MoneyPounds(previousFieldValue.toInt ,0).quantity</p>
```

Comment:
We should probably use this util here https://github.com/hmrc/tai-frontend/blob/master/app/uk/gov/hmrc/tai/util/MonetaryUtil.scala#L29

Reply:
Addressed